### PR TITLE
Release automation

### PR DESCRIPTION
**Pull request by kadel**
this automates binary uploading to github release page

it is documented in docs/contributing.md

this is how release draft that is cread by travis looks like:

screen shot 2018-02-13 at 17 45 31

all that we have to do is provide a description and click publish draft button

I've also created new bot account  www.github.com/ocdev-bot this account is used for uploading to github

### CODE REVIEW

**Comment by kadel** [comment]
prerequisite for https://github.com/redhat-developer/ocdev/issues/73

File: docs/contributing.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Contributing guide
+
+
+
+## making a release
```
Comment (containscafeine) [inline docs/contributing.md]: capitalize "M"

File: docs/contributing.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+Making artifacts for new release is automated. 
+When new git tag is created, Travis-ci deploy job automatically builds binaries and uploads it to Github release page.
```
Comment (containscafeine) [inline docs/contributing.md]: GitHub

File: docs/contributing.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+1. Crate PR with updated version in `cmd/version.go`
```
Comment (containscafeine) [inline docs/contributing.md]: Create

File: docs/contributing.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+2. When PR is merged create and push new git tag for version.
+    ```
+    git tag v0.0.1
+    git push upstream v0.0.1
+    ```
+    Or create new release using Github site (this has to be proper release not just draft).
```
Comment (containscafeine) [inline docs/contributing.md]: GitHub

File: docs/contributing.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+1. Crate PR with updated version in `cmd/version.go`
+2. When PR is merged create and push new git tag for version.
+    ```
+    git tag v0.0.1
+    git push upstream v0.0.1
+    ```
+    Or create new release using Github site (this has to be proper release not just draft). 
+    Do not upload any binaries for release
+    When new tag is created Travis-CI starts special deploy job.
```
Comment (containscafeine) [inline docs/contributing.md]: starts "a" special

File: docs/contributing.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    This job builds binaries automatically (via `make prepare-release`) and than uploads it to Github release page.
```
Comment (containscafeine) [inline docs/contributing.md]: s/than/then

Comment (containscafeine) [reply docs/contributing.md]: s/Github/GitHub

**Comment by kadel** [comment]
> LGTM, with a few nits in docs. Feel free to merge when resolved. 🎉

sorry for such horrible typos, my spell check failed me :-)

**Pull request merged**